friends/src/App.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import './App.css';
+
+import Friends from './components/Friends/Friends';
+import AddFriend from './components/AddFriend/AddFriend';
+
+const URL = "http://localhost:5000/friends";
+
+class App extends Component {
+  state = {
+    data : [],
+    name: '',
+    email: '',
+    age : 0,
+    id : 8,
+    updateID: 0,
+    update: false
+  }
+  componentDidMount(){
+    axios.get(URL)
+      .then(response => this.setState({
+        data: response.data
+      }))
+      .catch(err => console.log(err));
+  }
+
+  handleChange = event => {
+    this.setState({
+        [event.target.name] : event.target.value
+    });
+}
+  handleClick = event => {
+    event.preventDefault();
```

Comment:
good job remembering to use the event.preventDefault. Sometime people forget about, well done